### PR TITLE
Feat /  Tab to display user's Posts and blogs

### PR DESCRIPTION
**Pull request by cody-daigle**

### CODE REVIEW

**Review by mikesamm — APPROVED** [review]
Noice!

File: client/src/components/Blogs.tsx [hunk 1]
```diff
@@ -1,29 +1,46 @@
-import React, { useState, useEffect, ReactElement } from 'react';
+import React, { useState, useContext, ReactElement, useEffect } from 'react';
```
Comment (AlexPHebert2000) [inline client/src/components/Blogs.tsx]: You're not using `useContext` in the file

File: client/src/components/Login.tsx [hunk 2]
```diff
@@ -1,31 +1,8 @@
-
-import React, { ReactElement, useState } from 'react';
-import { Button, Box, Typography, TextField } from '@mui/material';
+import React, { ReactElement, useContext } from 'react';
```
Comment (AlexPHebert2000) [inline client/src/components/Login.tsx]: You're not using `useContext` in the file

File: client/src/components/Profile.tsx [hunk 3]
```diff
@@ -41,14 +43,21 @@ const Profile = (): ReactElement => {
   const [posts, setPosts] = useState<Post[]>([]);
   const [postsCount, setPostsCount] = useState<number>(0);
   const [followersCount, setFollowersCount] = useState<number>(0);
-  const [blogs, setBlogs] = useState<Blog[]>([]);
   const [username, setUsername] = useState<string>('');
-  const [value, setValue] = React.useState(0);
+  const [devId, setDevId] = useState<string>('');
+  const [githubId, setGithubId] = useState<string>('');
+  const [linkedinId, setLinkedinId] = useState<string>('');
```
Comment (AlexPHebert2000) [inline client/src/components/Profile.tsx:42]: Are you using this state `githubId` and `linkedinId`?

File: client/src/components/User.tsx
Comment (AlexPHebert2000) [inline client/src/components/User.tsx:1]: Are we using this context provider?

File: client/src/components/UserPosts.tsx [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import React, { ReactElement } from 'react';
+import React, { ReactElement, useEffect } from 'react';
```
Comment (AlexPHebert2000) [inline client/src/components/UserPosts.tsx]: You're not using `useEffect` in this file

File: client/src/components/UsersPost.tsx [hunk 5]
```diff
@@ -24,31 +29,90 @@ interface Post {
...
+    setEdit(true);
+  };
+
+  const handleTitleChange = (e: any) => {
+    setEditedPost({ ...editedPost, title: e.target.value });
+  };
+
+  const handleBodyChange = (e: any) => {
+    setEditedPost({ ...editedPost, body: e.target.value });
+  };
```
Comment (AlexPHebert2000) [inline client/src/components/UsersPost.tsx:57]: `handleBodyChange` and `handleTitleChange` can be combined into a single function see [PostCreationPage](./PostCreationPage.tsx) `handleTextInput`

Comment (AlexPHebert2000) [inline client/src/components/UsersPost.tsx:42]: `handleTitleChange` and `handleBodyChange` can be combined into a single function

**Review by AlexPHebert2000 — CHANGES_REQUESTED** [review]
See previous comments

File: client/src/components/User.tsx
Comment (cody-daigle) [reply client/src/components/User.tsx:1]: I removed it everywhere besides in the import. I'm going to correct these and then open the PR again

**Review by AlexPHebert2000 — APPROVED** [review]

**Pull request merged**